### DISCUSSION TITLE
Light curses theme

### DISCUSSION THREAD

**Discussion by lua-rocks** [orbitalquark/textadept#125] [Q&A] [answered]
I really love Textadept default light theme and I want to see it in terminal. Don't know why but I can't focus at all when working in blacks theme.

**Comment by orbitalquark** [comment, marked as answer]
Textadept's terminal version has no control over the colors your terminal outputs. It uses the existing palette. Try changing your terminal's color mapping to match Textadept's light theme, and then create your own *term.lua* theme that matches Textadept's light theme styles (not the colors).

**Reply by lua-rocks** [reply]
I got it, thank you!
